src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Display />
+      <ButtonPanel />
+    </div>
+  );
+}
```

Comment:
[OPTIONAL]

Consider using an arrow function introduced in ES6 instead. Arrows is a new syntax for functions, which brings several benefits:

- Arrow syntax automatically binds this to the surrounding code’s context
- The syntax allows an implicit return when there is no body block, resulting in shorter and simpler code in some cases
- Last but not least, => is shorter and simpler than function, although stylistic issues are often subjective.

Reply:
Implemented arrow functions, thanks for the heads up :+1: